src/components/SubTypeAccordion/SubTypeAccordionsContainer.js

```diff
@@ -2,41 +2,45 @@ import React from 'react';
 import {
   StyleSheet, View,
 } from 'react-native';
-import { connect } from 'react-redux';
-import { shape, arrayOf } from 'prop-types';
-import { selectedRecordsGroupedByTypeSelector } from '../../redux/selectors';
+import { shape, arrayOf, bool } from 'prop-types';
 
 import SubTypeAccordion from './SubTypeAccordion';
 
-const SubTypeAccordionsContainer = ({ selectedRecordsGroupedByType }) => (
+const SubTypeAccordionsContainer = ({ data, isDescending, fromDetailsPanel }) => (
   <View style={styles.root}>
     <View style={styles.container}>
-      { selectedRecordsGroupedByType
-        .map((typeObject) => (
-          <View key={typeObject.label}>
-            {typeObject.subTypes.map(({ subType, recordIds }) => (
-              <SubTypeAccordion
-                key={subType}
-                subType={subType}
-                resourceIds={recordIds}
-                subTypeCount={recordIds.length}
-              />
-            ))}
-          </View>
-        ))}
+      {data.map(({ subType, recordIds }) => {
+        // eslint-disable-next-line no-nested-ternary, max-len
+        const sortedRecordIds = !fromDetailsPanel
+          ? recordIds
+          : (isDescending ? recordIds : [...recordIds].reverse());
+
+        return (
+          <SubTypeAccordion
+            key={subType}
+            subType={subType}
+            resourceIds={sortedRecordIds}
+            subTypeCount={recordIds.length}
+            fromDetailsPanel={fromDetailsPanel}
```

Comment:
`fromDetailsPanel` is very particular, and I wonder if this should describe what it does, if possible
(`showOptionalIcons`, or whatever?)

Reply:
fromDetailsPanel does a few things downstream. it sets option icons for both the accordionHeader and RecordCard, but it also tells the Accordion if it's records can be reordered. Additionally, it'll be used in the future `DateAccordions` which can be reordered at the group level.